packages/alfa-rules/test/common/dom/get-colors.spec.tsx

```diff
@@ -289,6 +289,77 @@ test("getForeground() handles a mix of opacity and transparency and a linear gra
   });
 });
 
+test("getForeground() resolves `currentcolor` when color is set on parent", (t) => {
+  const target = <p>Content</p>;
+
+  h.document(
+    [
+      <html>
+        <div class="blue">
+          <div>
+            <div>
+              <div>
+                <div>
+                  <div>
+                    <div>
+                      <div>
+                        <div>Hello {target}!</div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </html>,
+    ],
+    [
+      h.sheet([
+        h.rule.style("p", {
```

Comment:
```suggestion
        h.rule.style("div", {
```
(actually, this is OK as it is, but the innermost `<div>` has `color: inherited` not `currentcolor`, so I'm not sure what the deep nesting test 🤔 )